src/tokio/unix.rs

```diff
@@ -13,4 +14,33 @@ impl AsyncCommandGroup for Command {
 
 		self.spawn().map(AsyncGroupChild::new)
 	}
+
+	fn group(&mut self) -> crate::builder::CommandGroupBuilder<tokio::process::Command> {
+		crate::builder::CommandGroupBuilder::new(self)
+	}
+}
+
+impl CommandGroupBuilder<'_, tokio::process::Command> {
+	/// Executes the command as a child process group, returning a handle to it.
+	///
+	/// By default, stdin, stdout and stderr are inherited from the parent.
+	///
+	/// On Windows, this creates a job object instead of a POSIX process group.
+	///
+	/// # Examples
+	///
+	/// Basic usage:
+	///
+	/// ```no_run
+	/// use std::process::Command;
+	/// use command_group::CommandGroup;
+	///
+	/// Command::new("ls")
+	///         .group()
+	/// 		.spawn()
+	///         .expect("ls command failed to start");
+	/// ```
+	pub fn spawn(&mut self) -> std::io::Result<AsyncGroupChild> {
+		self.command.group_spawn()
```

Comment:
To avoid code duplication I'd rather the inverse pattern if possible, with group_spawn() calling group().spawn(); it could then be a default implementation on the trait, requiring only group() and the builder to be implemented for a given platform/backing library

Reply:
Great idea, all done.